pkg/generators/openapi.go

```diff
@@ -453,23 +457,17 @@ func (g openAPITypeWriter) generateExtensions(CommentLines []string) error {
 }
 
 // TODO(#44005): Move this validation outside of this generator (probably to policy verifier)
-func (g openAPITypeWriter) validatePatchTags(m *types.Member, parent *types.Type) error {
-	patchMergeKeyStructTag, patchStrategyStructTag := getPatchTags(m)
-	patchMergeKeyCommentTag, err := getSingleTagsValue(m.CommentLines, tagPatchMergeKey)
-	if err != nil {
-		return err
-	}
-	patchStrategyCommentTag, err := getSingleTagsValue(m.CommentLines, tagPatchStrategy)
-	if err != nil {
-		return err
-	}
-	if patchMergeKeyStructTag != patchMergeKeyCommentTag {
-		return fmt.Errorf("patchMergeKey in comment and struct tags should match for member (%s) of (%s)",
-			m.Name, parent.Name.String())
-	}
-	if patchStrategyStructTag != patchStrategyCommentTag {
-		return fmt.Errorf("patchStrategy in comment and struct tags should match for member (%s) of (%s)",
-			m.Name, parent.Name.String())
+func (g openAPITypeWriter) validateTags(m *types.Member, parent *types.Type) error {
```

Comment:
Per our offiline discussion, please keep a temporary list of all tags need to be present in both comments and struct tags. Because the server is going to consume openapi spec back for apply, we do not need any of these tags in struct tags and comment tags should suffice.